spec/models/form/lettings/questions/wheelchair_spec.rb

```diff
@@ -39,4 +45,14 @@
   it "is not marked as derived" do
     expect(question.derived?).to be false
   end
+
```

Comment:
I think the existing test around hint text ought to be in a specific context too, since what we'll expect the default to be will change at some point.

Reply:
hm, I've stubbed `start_year_after_2024?: false` as a default, so the tests shouldn't start failing based on when we run them, or did you mean doing it purely for visibility?